File: Selenium/Activities/Activity19.java
package example;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity19 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/alerts");
		System.out.println("Title of the page is: " + driver.getTitle());
		driver.findElement(By.id("confirmation")).click();
		Alert alert = driver.switchTo().alert();
		System.out.println(alert.getText());
		alert.accept();
		driver.findElement(By.id("confirmation")).click();
		alert.dismiss();
		driver.quit();
		
		
		
	}

}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.common.by import By
import time


service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://training-support.net/selenium/javascript-alerts")
    print("Page Title :- " + driver.title)
    driver.find_element(By.ID,"confirm").click()
    alrt=driver.switch_to.alert
    print(alrt.text)
    alrt.accept()
    driver.find_element(By.ID,"confirm").click()
    alrt.dismiss()
    driver.close()

